lexicons/app/bsky/actor/defs.json

```diff
@@ -134,6 +134,16 @@
           }
         }
       }
+    },
+    "personalDetailsPref": {
```

Comment:
I think we may want to add this to the `#preferences` union a few dozen lines above.

Reply:
Oh shoot, right